README.md

```diff
@@ -22,14 +22,33 @@ Get help inforamtion for available commands,
 Initialize the dependencies by coping the holochain binaries (**Required**),
 
 ```shell
+# Initializes the ad4m-host with default config
 ./ad4m init
+
+# Initalize ad4m-host with different environment
+./ad4m init --dataPath environment_name --networkBootstrapSeed ./custom_seed.json
+
+# To override the environment you can use overrideConfig flag
```

Comment:
I don't quite get it here, do you mean the overrideConfig will override all the data identified by `environment_name`?

Reply:
yes

Reply:
@fayeed should we call it something like clean state?